review(tutorials): updated documentation MTA-2390 (#1009)
scaleway/docs-content: tutorials/jitsi-debian/index.mdx

    systemctl enable nginx.service
    ```

    <Message type="important">

>     If Nginx or Apache are not installed, Jitsi Meet will automatically install [Jetty](https://www.eclipse.org/jetty/) during the installation.

Reviewer: ```suggestion
    If you have not installed Nginx or Apache, Jitsi Meet will automatically install [Jetty](https://www.eclipse.org/jetty/) during the installation.
```

Reviewer: I'm not sure the indentation of the text inside the message box is correct, dit it display correctly when you did the build?

Reviewer: Yes perfect!

    ```
    apt install -y jitsi-meet
    ```

>     When asked, enter:

Reviewer: ```suggestion
    When asked:
```

>       - The FQDN of your instance. For example `jitsi.mydomain.tld` and press Enter.

Reviewer: ```suggestion
      - enter the FQDN of your Instance. For example `jitsi.mydomain.tld` and press Enter.
```

>       - When asked about the SSL certificate choose the `Generate a new self-signed certificate (You will later get a chance to obtain a Let's Encrypt certificate)` option and press Enter.

Reviewer: ```suggestion
      - about the SSL certificate, choose the `Generate a new self-signed certificate` (You will later get a chance to obtain a Let's Encrypt certificate) option and press Enter.
```

> 6. Run the following script to generate a [Let's Encrypt](https://letsencrypt.org) SSL certificate for your instance:

Reviewer: ```suggestion
6. Run the following script to generate a [Let's Encrypt](https://letsencrypt.org) SSL certificate for your Instance:
```

    ```
    /usr/share/jitsi-meet/scripts/install-letsencrypt-cert.sh
    ```

    <Message type="tip">
    If the generation of the Let's Encrypt certificate fails, run the following command: `dpkg-reconfigure jitsi-meet-web-config` to reconfigure the package `jitsi-meet-web-config` before relaunching the certificate request.
    </Message>

> 7. Open a Web-browser and type the FQDN of your instance, for example: `https://jitsi.mydomain.tld`. The following screen displays:

Reviewer: ```suggestion
7. Open a Web-browser and type the FQDN of your Instance, for example: `https://jitsi.mydomain.tld`. The following screen displays:
```
